errormessages/src/ts/main.tsx

```diff
@@ -0,0 +1,108 @@
+///**
+// * This Source Code Form is subject to the terms of the Mozilla Public
+// * License, v. 2.0. If a copy of the MPL was not distributed with this
+// * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// */
+
+/// <reference path="../tsd/tsd.d.ts" />
+
+import * as cu from 'cu-core';
+import events from 'cu-events';
+import * as React from 'react/addons';
+
+var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+declare const cuAPI: any;
+
+class ErrorMessage extends React.Component<any, any> {
+
+    render() {
+       
+        var commentNodes = this.props.messageList.map(function (comment: string, index: number) {
+            return (
+                <li className='message' key={index + comment}>{comment}</li>
```

Comment:
Nope just following an example, but its not needed, I'll pull it out no point to it.